crates/cli/src/opts/build/core.rs

```diff
@@ -126,6 +126,13 @@ pub struct CoreBuildArgs {
     #[serde(skip)]
     pub skip: Option<Vec<SkipBuildFilter>>,
 
+    /// Build files within specific directories.
+    ///
+    /// Child directories are not accounted and should be explicitly added.
+    #[arg(long, num_args(1..))]
+    #[serde(skip)]
+    pub dirs: Option<Vec<PathBuf>>,
```

Comment:
should we also support individual files?
maybe we do `files:`?

Reply:
yeah, since now it supports single files I renamed it to `paths`